lib/date.nit

```diff
@@ -0,0 +1,163 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Date manipulation
+module date
+
+# The main class Date (comming soon...)
+#
+# 	var d = new Date(2014, 01, 09)
+# 	assert d.to_s == "2014/01/09"
+#
+class Date
+	super Comparable
```

Comment:
Just noticed it, but why don't you redef the `OTHER` type to Date ?

That way you can get rid of the `isa Date` statement in `<`